spec.emu

```diff
@@ -328,6 +328,65 @@ contributors: Ron Buckton, Ecma International
         1. Return ~invalid~.
       </emu-alg>
     </emu-clause>
+
+    <ins class="block">
+    <emu-clause id="sec-static-semantics-containsthrow" type="sdo">
+      <h1>Static Semantics: ContainsThrowOnRight ( ): a Boolean</h1>
+      <dl class="header">
+      </dl>
+      <emu-grammar>ThrowExpression : `throw` UnaryExpression</emu-grammar>
+      <emu-alg>
+        1. Return *true*.
+      </emu-alg>
+      <emu-grammar>
+        UpdateExpression :
+          LeftHandSideExpression
+          LeftHandSideExpression `++`
+          LeftHandSideExpression `--`
+      </emu-grammar>
+      <emu-alg>
+        1. Return *false*.
+      </emu-alg>
+      <emu-grammar>
+        UpdateExpression :
+          `++` UnaryExpression
+          `--` UnaryExpression
+
+        UnaryExpression :
+          `delete` UnaryExpression
+          `void` UnaryExpression
+          `typeof` UnaryExpression
+          `+` UnaryExpression
+          `-` UnaryExpression
+          `~` UnaryExpression
+          `!` UnaryExpression
+
+        AwaitExpression :
+          `await` UnaryExpression
+      </emu-grammar>
+      <emu-alg>
+        1. Return ContainsThrowOnRight of |UnaryExpression|.
+      </emu-alg>

```

Comment:
This is not needed because of the implicit definition described at https://tc39.es/ecma262/#sec-algorithm-conventions-syntax-directed-operations

(I will review the rest properly later)